frontend/src/actions/parcelsActions.ts

```diff
@@ -71,6 +71,7 @@ export enum LeasedLand {
 export interface IBuilding extends IProperty {
   parcelId: number;
   address: IAddress;
+  administrativeArea: string;
```

Comment:
I'm not sure if this was the best way to do to access the fields, this was just the most obvious to me

Reply:
I believe that these are all valid fields returned by the API, so seems fine to me.